pkg/integration/prometheus/prometheus.go

```diff
@@ -0,0 +1,61 @@
+package prometheus
+
+import (
+	"github.com/k8sgpt-ai/k8sgpt/pkg/common"
+	"github.com/spf13/viper"
+)
+
+const (
+	ConfigValidate = "PrometheusConfigValidate"
+	ConfigRelabel  = "PrometheusConfigRelabelReport"
+)
+
+type Prometheus struct {
+}
+
+func NewPrometheus() *Prometheus {
+	return &Prometheus{}
+}
+
+func (p *Prometheus) Deploy(_ string) error {
+	// no-op
```

Comment:
We don't want to fallback to a Helm chart installation (as in the case of Trivy integration) if we don't detect Prometheus installed on the cluster, right?

Reply:
My understanding was that `integrations` manage workloads by default, and that you'd need to specify `--no-install` explicitly when activating to avoid this.

I feel like this is a slippery slope and I'd prefer not to have the tool manage third-party workloads in the cluster.

Especially the case for Prometheus, where most clusters probably have some form of Prometheus running anyway.

The downside of this is we need to make assumptions about the characteristics of the K8s resources in-source (as done here). However, that seems preferable, and ultimately more useful than relying on users managing the tools through `k8sgpt`.

Though this may be a larger discussion. I'm sure @AlexsJones has thoughts.

Reply:
I do agree having a bunch of `no-op` methods here isn't great 😒

Reply:
I think in this case as long as we document on `docs.k8sgpt.ai` how this works and maybe have a message printed it's fine e.g. "Activating Prometheus integration... Found existing installation" to help connect the dots for the user.. I do agree that the install pattern is not ideal - I did that out of necessity originally, but I am actually keen to remove it from Trivy

Reply:
Sounds good. Filed #858 as a follow-up. I can take that on next.